src/plugins/ColorVisionPlugin.js

```diff
@@ -0,0 +1,108 @@
+import { BasePlugin } from './BasePlugin';
+
+const COLOR_BLIND_TYPES = [
+  'none',
+  'protanopia',
+  'deuteranopia',
+  'tritanopia',
+  'achromatopsia'
+];
+
+/**
+ * @export
+ * @class ColorVisionPlugin
+ * @extends {BasePlugin}
+ */
+export class ColorVisionPlugin extends BasePlugin {
+  /**
+   *Creates an instance of ColorVisionPlugin.
+   * @param {object} params
+   * @param {string | HTMLElement} params.colorSelect
+   * @memberof ColorVision
+   */
+  constructor({ colorSelect }) {
+    super('Color-Filter-Plugin');
+
+    this.colorDropdown =
+      colorSelect instanceof HTMLSelectElement
+        ? colorSelect
+        : document.querySelector(colorSelect);
+
+    if (!this.colorDropdown) {
+      console.error(
+        'ColorVisionPlugin was not provided with a correct selector string'
+      );
+      return;
+    }
+
+    this.colorDropdown.innerHTML = '';
+  }
+
+  /**
+   * @memberof ColorVisionPlugin
+   */
+  onColorChange() {
+    this.sendProperty(
+      ColorVisionPlugin.colorVisionKey,
+      this.colorDropdown.value
+    );
+  }
+
+  /**
+   * @memberof ColorVisionPlugin
+   */
+  init() {
+    this.client.on(
+      'features',
+      function(features) {
+        if (!features.data) {
+          return;
+        }
+        if (!this.colorDropdown) {
+          return;
+        }
+        if (this.colorDropdown.tagName.toLowerCase() !== 'select') {
+          this.colorDropdown.style.display = 'none';
+          console.warn(
```

Comment:
I'm thinking this should be a `console.error` since it stops execution (due to the `return` below).

Reply:
Done :)